### DISCUSSION TITLE
[Weekly Review Report]: July 14 - 20

### DISCUSSION THREAD

**Discussion by eweitz** [wikipathways/wikipathways-help#124] [curation-reports]
### Recent changes

**By organism**
 * Homo sapiens: 147
 * Bos taurus: 15
 * Mus musculus: 9
 * Arabidopsis thaliana: 7
 * Rattus norvegicus: 4
 * Caenorhabditis elegans: 3
 * Oryza sativa: 2
 * Anopheles gambiae: 2
 * Saccharomyces cerevisiae: 1
 * Canis familiaris: 1
 * *Others: 1* 

**By author**
 * Eweitz: 119
 * Egonw: 73
 
**By description**
 * Removed template comments: 46
 * Economize layout: 19
 * Mitigate graphical discrepancy between PathVisio and Kaavio / Pvjs: 10
 * Modified description: 9
 * Removed a template comment: 9
 * Modified title: 5
 * Ontology Term : 'monocyte' added !: 3
 * Ontology Term : 'macrophage' added !: 3
 * Standardize case: 3
 * Economize layout, standardize case: 3
 * *Others: 77*

See [wikipathways-review.js](https://gist.github.com/eweitz/1dd6bff11eab74a525c86454563703af) for methodology to reproduce above report.  A record of the changes reviewed is available in [RecentPathwayChanges__WikiPathways_2024-07-22.txt](https://github.com/user-attachments/files/16332724/RecentPathwayChanges__WikiPathways_2024-07-22.txt).

### Under construction

 * All [21 "Under construction" tags](https://classic.wikipathways.org/index.php?title=Special:CurationTags&showPathwaysFor=Curation:UnderConstruction) were kept.

### New pathways

 * 0 new pathways

### Deleted Pathways

 * [8 pathways are tagged with curation tag: Request deletion.](https://classic.wikipathways.org/index.php?title=Special:CurationTags&showPathwaysFor=Curation:ProposedDeletion)

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

To see unapproved pathways, go to https://classic.wikipathways.org/index.php?title=Special:CurationTags&showPathwaysFor=Curation:AnalysisCollection and sort by last column.

**Comment by khanspers** [comment]
Thank you @eweitz for the fantastic report! 

I went through the pathways tagged for deletion and deleted the relevant ones. For WP5454, I removed the deletion tag, since the authors had selected the other version of this pathway, WP5448, for deletion. 

For WP4629/WP4628, I was originally going to deleted WP4629, after discussions with the author. This augmented version of the pathway was created for a thesis, with pathway elements referencing data from a specific paper. However, since this pathway is linked to from the thesis, I didn't want to remove it without first setting up a redirect, which I haven't done yet. For now, I will remove the deletion tag.
